rekuho/src/lib/Cursors.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts" context="module">
+  import type { Writable } from "svelte/store"
+  import { writable } from "svelte/store"
+
+  import type { Vector } from "./LinearAlgebra"
```

Comment:
Not quite of sure why we need "type" here but I trust you that we do

Reply:
Type aliases and class imports **must** be imported with a `import type ...` statement. Otherwise it will error with:
```
This import is never used as a value and must use 'import type' because 'importsNotUsedAsValues' is set to 'error'.ts(1371)
```